### PR TITLE
add the Niryo Ned2 to the robots list

### PR DESCRIPTION
**Pull request by jmottier-niryo**

### CODE REVIEW

File: _posts/2023-04-06-ned2.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+- 'education'
+- 'manipulator'
+- 'moveit'
+- 'noetic'
+- 'raspberry pi'
+- 'research'
+- 'training'
+twitter_text: Checkout the Ned2
+website: https://niryo.com/products-cobots/robot-ned-2/
+wiki_homepage:
```
Comment (tfoote) [inline _posts/2023-04-06-ned2.md]: Please add a link to ROS developer documentation here.

**Review by tfoote — CHANGES_REQUESTED** [review]
Please add links to ROS interface documentation.

File: _posts/2023-04-06-ned2.md (hunk 1 above)
Comment (jmottier-niryo) [reply _posts/2023-04-06-ned2.md]: Issue resolved with the commit bf7a2f2faf6e75e285ef8eed26220f9046ab1615

**Comment by jmottier-niryo** [comment]
@tfoote I allow myself to tag you in case you missed this pull request update

**Review by tfoote — APPROVED** [review]
Thanks for the ping, please do that after making an update. Reviewers don't get a notification when new changes are pushed. 

The new link looks good.

**Pull request merged**